apps/artifactory/sbox-intsvc/base/kustomization.yaml

```diff
@@ -7,4 +7,7 @@ resources:
   - ../admin-pw.yaml
 
 patches:
-  - path: ../ingress-patch.yaml
\ No newline at end of file
+  - path: ../ingress-patch.yaml
+
+patchesStrategicMerge:
+    - ../../artifactory/artifactory-sbox.yaml
```

Comment:
Why are you trying to use both here now? Can you just add the sbox file as another patch please